src/components/ui/loader.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentPropsWithoutRef } from 'react';
+
+import { cn } from '../../lib/utils';
+
+type LoaderProps = ComponentPropsWithoutRef<'div'>;
+
+export const Loader = ({ className }: LoaderProps) => {
+  return (
+    <div
+      className={cn(
+        'h-4 w-4 animate-spin rounded-full border-t-2 border-t-white',
```

Comment:
https://tailwindcss.com/docs/size you can replace h-4 w-4 by size-4

Reply:
Thanks for the tip! I didn’t know about that Tailwind shortcut.